### DISCUSSION TITLE
Why there are some invalid plots?

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#6871] [Support]
I have more than 3PiB now and I always find there are some invalid plots in my plot list.
I didn't pay much attention to it since it just very few parts in all of my plots.
However, I wondering why there are invalid plots and what causes them invalid.
Currently, I got 26 pages of invalid plots which are about 260 plots out of 39k plots in total.
I have checked the invalid plots and it says there are badbit or failbit somewhere. The invalid plots are usually smaller than the valid plots, something like 99.7GB.
I checked my plotters and everything is fine. Are there any reasons why that I got invalid plots?

**Comment by loppefaaret** [comment]
that seems to indicate they are missing the phase 4 data ? perhaps it was during copy they lost data ? copy to a filesystem with not enough room ?

**Comment by BrandtH22** [comment]
@cccat6 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
